### PR TITLE
Fixes 6223 graph button indistinguishable (#6223). Replaces Button with FloatingButton

### LINKED ISSUE #6223: Graph Button in Data Model Settings Blends with Background
# Current behavior

In `Settings > Data model`, the Graph button is indistinguishable from the background.

CleanShot 2024-07-11 at 17 35 55

# Desired Behavior

Use the Floating button instead:

https://storybook.twenty.com/?path=/story/ui-input-button-floatingbutton--catalog

### PR DESCRIPTION
**Pull request by Faisal-imtiyaz123**
fixes #6223

### CODE REVIEW

File: packages/twenty-front/src/modules/ui/input/button/components/FloatingButton.tsx [hunk 1]
```diff
@@ -87,6 +94,7 @@ const StyledButton = styled.button<
   &:focus {
     outline: none;
   }
+  text-decoration: ${({ to }) => to && 'none'};
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/ui/input/button/components/FloatingButton.tsx:97]: text-decoration should actually always be 'none', let's remove 'to' from this condition and from the styledButton signature

**Review by charlesBochet — COMMENTED** [review]
Thanks @Faisal-imtiyaz123 for the contribution! Looks good, just a small change :)

**Review by charlesBochet — COMMENTED** [review]
Left a last comment!

File: packages/twenty-front/src/modules/ui/input/button/components/FloatingButton.tsx [hunk 2]
```diff
@@ -16,12 +16,19 @@ export type FloatingButtonProps = {
...
   Pick<
     FloatingButtonProps,
-    'size' | 'focus' | 'position' | 'applyBlur' | 'applyShadow' | 'position'
+    | 'size'
+    | 'focus'
+    | 'position'
+    | 'applyBlur'
+    | 'applyShadow'
+    | 'position'
+    | 'to'
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/ui/input/button/components/FloatingButton.tsx:31]: no need to pass the 'to' here anymore

Comment (Faisal-imtiyaz123) [reply packages/twenty-front/src/modules/ui/input/button/components/FloatingButton.tsx:31]: ok

**Comment by charlesBochet** [comment]
My bad, we actually need the 'to' to be there. I missed the 'as' props that use Link in this case

**Review by charlesBochet — APPROVED** [review]
LGTM, thank you!

**Pull request merged**